src/ctia/bundle/routes.clj

```diff
@@ -67,9 +67,28 @@
     :read-casebook
     :list-casebooks})
 
+(defn- keyword->str [key] (str (if (namespace key) "/" "") (name key)))
+
+(defn- entity->bundle-keys
+  "For given entity key returns corresponding keys that may be present in Bundle schema.
+  e.g. :asset => [:assets :asset_refs]"
+  [entity-key]
+  (let [{:keys [entity plural]} (get (entities/all-entities) entity-key)
+        kw->snake-case-str      (fn [kw] (-> kw keyword->str (str/replace #"-" "_")))]
```

Comment:
We should have only simple keywords in bundle, you can remove `keyword->str`.
```suggestion
        kw->snake-case-str      (fn [kw] (-> kw name (str/replace #"-" "_")))]
```

Reply:
Initially when I added `keyword->str`, I thought maybe we can later move it into some `utils` namespace and re-use, but you're right, it's YAGNI. I'm gonna get rid of it.